VadimVildanov/HW3/Task 3.py

```diff
@@ -0,0 +1,8 @@
+# Interchange the values
+first = 4
+second = 53
+
+print(first * second, second // first, first ** second)
```

Comment:
look at the right answer

first = 4
second = 53

first, second = second, first

print(first, second)

Reply:
I'm fixed bag)